### DISCUSSION TITLE
Play requires akka-serialization-jackson (not documented)

### DISCUSSION THREAD

**Discussion by skestle** [playframework/playframework#11378] [General]
I've updated my akka dependencies, and when running I get akkka mismatch exceptions related to `akka-serialization-jackson` (old) vs `akka-actor` etc.

This is not reflected on https://www.playframework.com/documentation/2.8.x/ScalaAkka

Can the list of akka dependencies be updated?
(I'll come back at some point once I've verified that this is the only mismatched dependency - it's the only one listed in akka's version message)

**Comment by mkurz** [comment]
Sorry a bit late, however, which Play version were/are you using and to from to which akka version did you upgrade?

**Comment by skestle** [comment]
I was trying to go from akka 2.5..... to 2.6.19. I ended up with play 2.8.16.

The issue was that the module I was depending on did not import all of akka, resulting in a akka-jackson version error - presumably since play pulls it in (hence this ticket)

All [I think] I'm asking for is for `akka-serialisation-jackson` to be documented in the dependencies section on the noted page.

What would be lovely though is a dependencies project like https://github.com/lightbend/akka-dependencies (Much of my development work is done on offline systems)
